lib/neo-test-functions.js

```diff
@@ -51,6 +51,32 @@
     return particle
   }
 
+  // Get the index of the AC
+  // This is the case with multiple ACs
+  function getAcIndex(path) {
```

Comment:
Same with this helper function: If we don't reuse it in other cases, we can keep it inside the `getAcData` function.

Reply:
Same response as :+1: : https://github.com/neohelden/test-automation/pull/6#discussion_r618313390